carbon-design-system/carbon
# test: increase coverage on `FilterableMultiSelect`

Closes #17479 

adding tests to increase coverage 

#### Changelog

**New**

- added new test cases covering various stuff related to downshift

#### Testing / Reviewing

- pull the PR down
- run the command `yarn test FilterableMultiSelect-test.js --coverage`
- check that coverage is above 85
